website/settings/defaults.py

```diff
@@ -69,6 +69,7 @@ def parent_dir(path):
 CONFIRM_REGISTRATIONS_BY_EMAIL = True
 ALLOW_REGISTRATION = True
 ALLOW_LOGIN = True
+ENABLE_NOTIFICATION_SUBSCRIPTION_CREATION = True
```

Comment:
Minor: This is a bit of a misnomer; in addition to subscribing a user on project creation, users are also subscribed on when they are added as a contributor. How about something like `SUBSCRIBE_CONTRIBUTORS_TO_NOTIFICATIONS`?